core/src/main/java/org/infinispan/manager/EmbeddedCacheManager.java

```diff
@@ -319,5 +320,11 @@ default ClusterExecutor executor() {
       throw new UnsupportedOperationException();
    }
 
-
+   /**
+    * Returns an entry point for a Health Check API.
+    *
+    * @since 9.0
+    * @return Health API for this {@link EmbeddedCacheManager}.
```

Comment:
I'd leave out the `@return` tag, it just repeats the sentence above. And I'd also leave out "Returns" from the sentence above :)